oauth2.py

```diff
@@ -366,9 +369,17 @@ def create_tweet_payload(text, in_reply_to_tweet_id=None):
                     self.platform,
                     response_data,
                 )
+                store_publication(
+                    country_code="Unknown",
+                    platform_name=self.platform,
+                    source="Platforms",
+                    gateway_client="Unknown",
+                    status="failed",
+                )
                 return response_data
 
             tweet_id = response.json()["data"]["id"]
+            status = "published"  
```

Comment:
No need to declare and assign the `status` variable here, since we can pass the values directly to the `store_publication` function argument.